src/languages/es.ts

```diff
@@ -1913,8 +1913,8 @@ export default {
             customers: 'Clientes/Proyectos',
             displayedAs: 'Mostrado como',
             accountsDescription: 'Los planes de cuentas se importan como categorías cuando está conectado con una integración de contaduría, esto no se puede desactivar.',
```

Comment:
maybe you forgot to update this translation?

Reply:
Don't have the Spanish understanding, but when I pasted this key in Google translate, it showed the old English copy.

Reply:
Lol, I guess that shows from where the Spanish translation came from. I'll see if I can come up with something closer to the current english translation.

Reply:
I updated the translation